_targets/build.project.armv7m7-imxrt117x

```diff
@@ -67,13 +114,15 @@ b_build_target() {
 	b_log "Building plo"
 	make -C plo clean all
 
-	cp "${PREFIX_PROG_STRIPPED}plo-ram-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.img" _boot/
-	cp "phoenix-rtos-hostutils/psu/plo-ram-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.sdp" _boot/
-
-	b_mkscript
+	MAGIC_USER_SCRIPT=$((($RANDOM % 0xffff) | (($RANDOM % 0xffff) << 16)))
```

Comment:
I don't know, if I like MAGIC being random... I would rather have something that doesn't change between builds

Reply:
- `$` not needed before `RANDOM` inside `$(())`
- I agree that building should be repeatable - building from the same tag/commit should always produce binary with the same crc32 (which is not the case in our buildsystem currently, but it might be desirable for MID certifications, etc.).

Reply:
After building plo, there are two images: 
- `plo_ram.img` - images is loaded to ram and is used to flash phoenix.disk
- `plo.img` - image which is a part of phoenix.disk

Both of the images contain preinit script in `.data` section with command: `call <dev> <script name> <magic>`. If you flash your memory for the first time, it's ok, commad `call` is not executed because `MAGIC` is incorrect. However, if you want to re-flash memory with the `MAGIC` defines as constant, execution of `call` will be fine. The side effect will be loading kernel to ram, where is currently plo.

I have discussed this approach with @pawelpisarczyk. The other way is to create the separate script for `plo_ram.img` without `call` command.

Reply:
Wouldn't it be better to have separate script for plo_ram.img? It is used only to program img after all

Reply:
I'm getting` Magic number for user.plo is wrong.` when loading plo to the RAM multiple times, but it doesn't really change anything...

Reply:
It is expected behavior. If You don't have random `MAGIC`, the command `call` will start execution of script located in flash. After that the kernel will be loaded to `ITCM` and plo will be overwritten and the application will crash.
So that I think that using random number in this case changes something ;)